docs/faq.rst

```diff
@@ -27,6 +27,18 @@ If you are using PyPy, we do not currently ship ``cryptography`` wheels for
 PyPy. You will need to install your own copy of OpenSSL -- we recommend using
 Homebrew.
 
+Compiling ``cryptography`` on macOS produces a ``fatal error: 'openssl/opensslv.h' file not found`` error
+---------------------------------------------------------------------------------------------------------
+
+You may get this message with the newest version of pip and a version of Python
+that can't install from wheels (like PyPy). Install OpenSSL using Homebrew
+(``brew install openssl``) and then link against Homebrew's OpenSSL when
+installing cryptography by running
```

Comment:
I think it might be be better to just link to the write sectino of our installation docs.